docs/sphinx/source/user_guide/extras/nomenclature.rst

```diff
@@ -141,10 +149,15 @@ There is a convention on consistent variable names throughout the library:
         Diode saturation current
 
     solar_azimuth
-        Azimuth angle of the sun in degrees East of North
+        Azimuth angle of the sun in degrees East of North. The solar azimuth angle
+        describes the sun’s position along the horizon relative to a reference point.
+        The pvlib-python convention is defined as degrees east of north, so
+        North = 0°, East = 90°, South = 180°, West = 270°.
 
     solar_zenith
-        Zenith angle of the sun in degrees
+        Zenith angle of the sun in degrees. This is the angle between the sun's rays and
```

Comment:
Nit: the angle is between a vector pointed straight up and a vector pointed at the sun, from the observer.